.travis.yml

```diff
@@ -17,7 +17,9 @@ env:
     - ROS_DISTRO=melodic
     - ROS_REPO=ros
     - UPSTREAM_WORKSPACE=moveit.rosinstall
-    - TEST_BLACKLIST=moveit_ros_perception  # mesh_filter_test fails due to broken Mesa OpenGL
+    # moveit_ros_perception: mesh_filter_test fails due to broken Mesa OpenGL
+    # moveit_experimental: test succeeds locally, not on Travis
```

Comment:
out of curiosity, why? usually people only run tests on Travis, so this is an important TODO in the future. can you create an issue for it, please?

Reply:
It may be a timing issue. The test is timing-based, like: check that X +/-1 messages were received in one seconds. Yep, will make an issue